src/common.cpp

```diff
@@ -232,20 +231,20 @@ namespace irods::http
         log::debug("{}: Bearer token: [{}]", __func__, bearer_token);
 
         // Verify the bearer token is known to the server. If not, return an error.
-        const auto* object = irods::process_stash::find(bearer_token);
-        if (!object) {
-            log::error("{}: Could not find bearer token matching [{}].", __func__, bearer_token);
+        const auto object{irods::http::process_stash::find(bearer_token)};
```

Comment:
Since we are touching these lines anyway, let's make the tabs and spaces consistent in this area so it's a little easier to read

Reply:
Can do, but I think the plan at some point was that @korydraughn was going to do a formatting sweep on the entire repo?

I remember the formatting was a bit odd on the OIDC code too...

Reply:
Yes, as long as it's about right, that's fine - there will be a full sweep prior to release.

Reply:
Yep, just took a little more brain power than usual to keep the scopes straight in my head ;)